06-building-a-program-start-to-finish/writing-tests/Output/tests.js

```diff
@@ -0,0 +1,190 @@
+const books = require('../books.json');
+const gotBooks = require('../got_books.json');
+const emptyArray = require('../empty_book.json');
+const invalidContent = require('../invalid_content.json');
+const invalidKey = require('../invalid_book.json');
+
```

Comment:
A base filename should exactly match the name of its default export. Please refer to your [Airbnb Style guide](https://github.com/airbnb/javascript#naming--filename-matches-export)